src/main/java/Figure.java

```diff
@@ -0,0 +1,5 @@
+public abstract class Figure {
+    public abstract String draw ();
```

Comment:
```suggestion
    public abstract String draw();
```
Використовуй комбінацію клавіш: Ctrl + Alt + L